api/v1/webspherelibertyapplication_types.go

```diff
@@ -429,6 +432,16 @@ type WebSphereLibertyApplicationRoute struct {
 	InsecureEdgeTerminationPolicy *routev1.InsecureEdgeTerminationPolicyType `json:"insecureEdgeTerminationPolicy,omitempty"`
 }
 
+type WebSphereLibertyApplicationSemeruCloudCompiler struct {
+	// SemeruCloudCompiler Resources
+	// +operator-sdk:csv:customresourcedefinitions:order=48,type=spec,displayName="Resource Requirements",xDescriptors="urn:alm:descriptor:com.tectonic.ui:resourceRequirements"
+	Resources *corev1.ResourceRequirements `json:"resources,omitempty"`
+
+	// Flag to disable JIT compiler
```

Comment:
The parent element (semeruCloudCompiler) as you have it below can determine whether to deploy JITServer or not. We don't need to disable JIT at the application level. So this can be removed.